src/components/data-processing/ConfigureEmbedding/CalculationConfig.jsx

```diff
@@ -195,8 +206,19 @@ const CalculationConfig = (props) => {
             onPressEnter={(e) => e.preventDefault()}
             onBlur={(e) => setPerplexity(e.target.value)}
           />
+          <Tooltip title='This determines how to much emphasis should be on local or global aspects of your data.
+          The parameter is, in a sense, a guess about the number of close neighbors each cell has.
+          In most implementations, perplexity defaults to 30. This focuses the attention of t-SNE on preserving the
+          distances to its 30 nearest neighbors and puts virtually no weight on preserving distances to the remaining points.
+          The perplexity value has a complex effect on the resulting pictures.'>
+            <QuestionCircleOutlined />
+          </Tooltip>
         </Form.Item>
+
         <Form.Item label='Learning rate'>
+
```

Comment:
Nitpick: could you remove these 3 empty spaces?

Reply:
done